src/components/EnterpriseContractView/useEnterpriseContractResultFromLogs.tsx

```diff
@@ -18,81 +15,39 @@ export const useEnterpriseContractResultFromLogs = (
 ): [ComponentEnterpriseContractResult[], boolean] => {
   const { namespace, workspace } = useWorkspaceInfo();
   const [taskRun, loaded, error] = useTaskRuns(namespace, pipelineRunName, 'verify');
-  const [fetchTknLogs, setFetchTknLogs] = React.useState<boolean>(false);
   const [ecJson, setEcJson] = React.useState<EnterpriseContractResult>();
   const [ecLoaded, setEcLoaded] = React.useState<boolean>(false);
-  const ecResultOpts = React.useMemo(() => {
-    const podName = loaded && !error ? taskRun[0].status.podName : null;
-    return podName
-      ? {
-          ns: namespace,
-          name: podName,
-          path: 'log',
-          queryParams: {
-            container: 'step-report',
-            follow: 'true',
-          },
-        }
-      : null;
-  }, [loaded, error, taskRun, namespace]);
 
   React.useEffect(() => {
-    let unmount = false;
-    if (ecResultOpts) {
-      commonFetchText(getK8sResourceURL(PodModel, undefined, ecResultOpts))
-        .then((res) => {
-          if (unmount) return;
-          const json = YAML.load(res) as EnterpriseContractResult;
-          setEcJson(json);
-          setEcLoaded(true);
-        })
-        .catch((err) => {
-          if (unmount) return;
-          if (err.code === 404) {
-            setFetchTknLogs(true);
-          } else {
-            setEcLoaded(true);
-          }
-          // eslint-disable-next-line no-console
-          console.warn('Error while fetching Enterprise Contract result from logs', err);
-        });
```

Comment:
This is required. When the pipeline run is running we fetch logs from Pod.
Below `getTaskRunLog` fetches the logs from tkn results.

Reply:
Am I wrong in my assumption that the logs are available on the "Logs" tab?

To me it seems that the `PipelineRunLogsTabProps` using `PipelineRunLogs` is responsible for that and this should only affect the data for the "Security" tab.

Reply:
Yes, logs are available only on the "Logs" tab. And for the security tab, it's the same mechanism as the logs tab if the pipeline is running, we use the pods to fetch the logs (i.e. results yaml) and parse the data for the security tab.
We don't have to wait for the pipeline run to finish to show the data for the security tab.

Reply:
I guess this relies on parsing partial YAML? This is an odd requirement for me, the logs from the `step-report` will be available after the main bulk of the work is done and milliseconds apart from the logs of the `report-json`. I do not think there is much benefit to the end user in doing this.

Reply:
In that case, I agree, We can remove that the logic to fetch the pod logs